src/zgemm_TN.jdf

```diff
@@ -169,8 +169,8 @@ BODY [type=CUDA
       B.size=%{ return descB->mb*descB->nb*parsec_datadist_getsizeoftype(descB->mtype);%}
       C.size=%{ return descC->mb*descC->nb*parsec_datadist_getsizeoftype(descC->mtype);%}
       A.dc=ddescA B.dc=ddescB C.dc=ddescC
-      stage_in=stage_in_lapack
-      stage_out=stage_out_lapack]
+      stage_in=dplasma_cuda_lapack_stage_in
+      stage_out=dplasma_cuda_lapack_stage_out]
```

Comment:
Why there is no HIP chore for the normal GEMM (aka. not the summa version) ?

Reply:
Not all kernels are implemented https://github.com/ICLDisco/dplasma/issues/98